test/database/task.test.ts

```diff
@@ -0,0 +1,32 @@
+import { assertRejects } from '$std/assert/assert_rejects.ts';
+// import { failCommit } from './dbUtils.ts';
+// import type { StoredTaskSchema } from '../../schema/task.ts';
+import { Task } from '../../database/task.ts';
+import { userId } from '../config.ts';
+
+Deno.test('Task', async (t) => {
+  // let task: StoredTaskSchema | undefined = undefined;
+
+  await t.step('create', async () => {
+    //@ts-ignore: testing for bad inputs
+    await assertRejects(() => Task.create({}));
+
+    task = await Task.create({
```

Comment:
This needs a `const` to make the linter happy.  May as well check that it returned something with `assert(task)` afterward.